tools/justfile

```diff
@@ -0,0 +1,136 @@
+# Enable positional args
+set positional-arguments
```

Comment:
I am not sure this is required.

Reply:
Yes, its false by default https://github.com/casey/just#table-of-settings

Reply:
Oh, I thought `*args` in a rule would be enough.